documentation/theme/tutorial-theming-overlay.asciidoc

```diff
@@ -34,63 +34,54 @@ Attributes from the main component are not copied to the overlay component, e.g.
 
 The theme attribute is an exception to the above, and it is copied to the overlay component, allowing styling of individual overlays.
 
-To style an overlay component it is necessary to create a style/theme module, in the same way that is done for other components, and target the stylable parts, optionally scoped with a theme variant selector:
+To style an overlay component it is necessary to create a style CSS module, target the stylable parts, instruct flow to import it
```

Comment:
There are couple of this kind of line breaks in the patch. Our convention is currently to use softwrap and one sentence per line in asciidoctor files -> makes it easier to review changes. Can you change this an the other parts as well.